RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -188,13 +188,7 @@ extension CloudBackupClient {
 
 			guard shouldBackUp || shouldReclaim else { return }
 
-			var profileToUpload = profile
-			if shouldReclaim {
-				// The profile will already be locally claimed, but we want to update the lastUsedOnDevice date
-				await profileStore.claimOwnership(of: &profileToUpload)
-			}
-
-			try? await backupProfileAndSaveResult(profileToUpload, existingRecord: existingRecord)
+			try? await backupProfileAndSaveResult(profile, existingRecord: existingRecord)
```

Comment:
good catch forgot to stage the claiming code.
The claim for this case does not happen in Sargon, since local Profile by definition claimed. It is needed to update the profile from the cloud to point to current device.